tests/ha/ha_install.pm

```diff
@@ -0,0 +1,46 @@
+# SUSE's SLES4SAP openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Install HA
+# Maintainer: QE-SAP <qe-sap@suse.de>
+
+use base 'sles4sap';
+use testapi;
+use utils;
+use serial_terminal 'select_serial_terminal';
+use power_action_utils 'power_action';
+use strict;
+use warnings;
+
+sub run {
+    my ($self) = @_;
+
+    select_console 'root-console';
+
+    # Workaround for current agama installation: registration, extensions and so on
+    # Install suseconnect-ng
+    zypper_call('in suseconnect-ng');
+    # De-register SLES
+    assert_script_run('suseconnect -d');
+    # Regster SLES
+    assert_script_run('SUSEConnect' . ' -r ' . get_required_var('SCC_REGCODE') . ' --url ' . get_required_var('SCC_URL'));
+    # Regster HA
+    assert_script_run('SUSEConnect' . ' -p ' . 'sle-ha/' . get_required_var('VERSION') . '/' . get_required_var('ARCH') . ' -r ' . get_required_var('SCC_REGCODE_HA'));
+    # Check HA extension
+    assert_script_run('suseconnect --status-text');
+    assert_script_run('suseconnect --list-extensions');
```

Comment:
Do we like to search something in the output?

Reply:
The original purpose is just for `debug info` as SLE16 is still not good. Deleted.